frontend/src/components/layout/navigation/AppPicker.tsx

```diff
@@ -31,16 +31,15 @@ import { OverlayProps } from "@react-aria/overlays";
 import { useLocalization } from "@fluent/react";
 import { event as gaEvent } from "react-ga";
 import styles from "./AppPicker.module.scss";
-import BentoIconGrey from "../../../../static/images/bento.svg";
-import BentoIconWhite from "../../../../static/images/bento-white.svg";
```

Comment:
Good riddance to these two - could you also remove the respective images? I don't think they're used anymore.